certbot/plugins/selection.py

```diff
@@ -39,6 +39,33 @@ def pick_authenticator(
     return pick_plugin(
         config, default, plugins, question, (interfaces.IAuthenticator,))
 
+def get_unprepared_installer(config, plugins):
+    """
+    Get an unprepared interfaces.IInstaller object.
+
+    :param certbot.interfaces.IConfig config: Configuration
+    :param certbot.plugins.disco.PluginsRegistry plugins:
+        All plugins registered as entry points.
+
+    :returns: Unprepared installer plugin or None
+    :rtype: IPlugin or None
+    """
+
+    _, req_inst = cli_plugin_requests(config)
+    if not req_inst:
+        return None
+    installers = plugins.filter(lambda p_ep: p_ep.name == req_inst)
```

Comment:
I'm sorry I didn't catch this in my previous review, but there's actually no check in this function that the plugin returned here is an installer. Can we add this?

I think we should do this by calling the `verify` method on the `PluginsRegistry` object like we do in `pick_plugin`. We should do this before calling `len(installers)` for consistency and to avoid erroring out if there's only one installer plugin with that name.

Reply:
Oh, nice catch! This is now added.